### ISSUE TITLE
BZ bot caught in a loop

### ISSUE THREAD

**Issue by Zlopez** [fedora-infra/the-new-hotness#494]
@FaramosCZ: 
Hi,
Take a look at this BZ: https://bugzilla.redhat.com/show_bug.cgi?id=2090416

Single new upstream release. But the BZ bot 'upstream release monitoring' submits the same patch again and again, trying new scratch builds again and again ... spamming people in CC in the process.

I would expect the bot to try once and give up instead.

**Comment by Zlopez** [comment]
I'm not sure what exactly is happening here, this seems like there is only one message from Anitya that was handled on the start and only the scratch build is started, which doesn't make much sense, I need to investigate this more closely to find out what could cause this.

**Comment by Zlopez** [comment]
So I tried to dig into this issue. It seems that this is happening only for this project, the main reason is the wrong setup of the project in [Anitya](https://release-monitoring.org/project/16939/), so I fixed that. The second issue is that 3.3.1 still isn't in Fedora, so every time there is a new version retrieved (even if it is a version that belongs to 2.*) it will try to start the scratch build for 3.3.1 because it's the latest one and it still isn't in the Fedora (the latest one is [3.2.7](https://src.fedoraproject.org/rpms/mariadb-connector-c)).

Only thing that the-new-hotness did wrong was the missing notification about newly retrieved version on Anitya, which I couldn't really found why this comment was missing. I tried to even check other packages for similar issue, but didn't found any.

The issue should be now fixed, because the project is now setup correctly and the bogus versions removed.
